src/webextension/widgets/toolbar.css

```diff
@@ -10,7 +10,7 @@
 }
 
 .toolbar > * + * {
-  margin-inline-start: 5ch;
+  margin-inline-start: 2.5ch;
```

Comment:
Wasn't there another PR I reviewed where we ended up changing this to px? I think we can drop this change.